.travis.yml

```diff
@@ -1,17 +1,44 @@
 language: julia
 os:
   - linux
+  - osx
+  - windows
```

Comment:
Not sure what this accomplishes other than to slow down CI … this is a pure-Julia package where portability hasn't been an issue.

Reply:
I don't have a strong opinion about them and I removed these lines, even though it happened to me in the past to find platform-dependent Julia quirks in pure-Julia packages